test/InitDelegateeDeposits.t.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Let's move this file into a new `test/script` subdir. We're doing something similar on the Staker repo right now.